database/insert_data.sh

```diff
@@ -2,17 +2,13 @@
 #insert data from
 echo -e "\nInserting data..."
 
-node ./database/insertData.js --from_insert_data.sh $1 &
+yarn ts-node ./database/insertDataCLI.ts $1 &
 
 # Makes script wait until async node script has completed
 PID=$!
 wait $PID
 
-# Turn on Row-level Security
-echo -e "\nEnabling Row-level Security..."
-psql -U postgres -q -b -d tmf_app_manager -f "./database/turn_on_row_level_security.sql"
-
-node ./database/updateRowPolicies.js --from_insert_data.sh &
+yarn ts-node ./database/updateRowPolicies.ts &
```

Comment:
Should us updateRowPoliciesCLI.ts

Reply:
323d4f9e47bba89b67e9c8cebad868c9f51f88c7